init.c

```diff
@@ -1253,6 +1253,15 @@ static enum CommandResult parse_ifdef(struct Buffer *buf, struct Buffer *s,
     res = !!myvar_get(buf->data);
   }
 
+  /* or an environment variable? */
+  if (!res)
+  {
+    if (mutt_str_getenv(buf->data) != NULL)
+    {
+      res = true;
+    }
```

Comment:
Actually, we can reduce this to your original suggestion:
```c
res = !!mutt_str_getenv(buf->data);
```

`mutt_str_getenv()` returns a pointer which we `!!` **not** twice (a common C trick to boolify a pointer).

Reply:
I've seen that syntax around but didn't know what it does. It's up to you and I'm all for it. It's shorter and it's consistent with my_ vars condition above.